var/spack/repos/builtin/packages/readline/package.py

```diff
@@ -19,6 +19,7 @@ class Readline(AutotoolsPackage, GNUMirrorPackage):
     gnu_mirror_path = "readline/readline-8.0.tar.gz"
 
     version("8.2", sha256="3feb7171f16a84ee82ca18a36d7b9be109a52c04f492a053331d7d1095007c35")
+    version("8.1.2", sha256="7589a2381a8419e68654a47623ce7dfcb756815c8fee726b98f90bf668af7bc6")
```

Comment:
I recently removed this version since it's included in 8.2 + 001 and 002 patches

Reply:
Ok, I dropped this from the change set.